auslib/web/admin/swagger/api.yaml

```diff
@@ -376,6 +376,34 @@ paths:
         '404':
           $ref: '#/responses/resourceNotFound'
 
+  /scheduled_change/diff/release/{sc_id}/{field}:
```

Comment:
There's no other field that's actually useful to diff - it would be better to explicitly only support "data" here.

Please add a test or two for this endpoint, too!